MarkupsToModel/Logic/vtkSlicerMarkupsToModelLogic.cxx

```diff
@@ -333,23 +351,33 @@ void vtkSlicerMarkupsToModelLogic::UpdateOutputCloseSurfaceModel( vtkMRMLMarkups
 
   vtkSmartPointer< vtkPolyData > cleanedPolyData = vtkSmartPointer< vtkPolyData >::New();
   cleanedPolyData = cleanPointPolyData->GetOutput();
```

Comment:
This deletes the vtkPolyData object that you created in the previous line (so you should not have created the new vtkPolyData in the previous line) - but anyway, you'll replace all this with a simple pointer.